theseus/src/api/settings.rs

```diff
@@ -49,10 +49,19 @@ pub async fn set(settings: Settings) -> crate::Result<()> {
     }
     .await;
 
+    let updated_discord_rpc = {
+        let read = state.settings.read().await;
+        settings.disable_discord_rpc != read.disable_discord_rpc
+    };
+
     {
         *state.settings.write().await = settings;
     }
 
+    if updated_discord_rpc {
+        state.discord_rpc.clear_to_default(true).await?;
+    }
+
```

Comment:
one thing I would test is to make sure discord rpc doesn't interfere with offline mode. i was seeing some issues with it

Reply:
was this tested?